osf/management/commands/clear_user_session_maps.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import django
+django.setup()
+
+from framework.celery_tasks import app as celery_app
+from osf.models import UserSessionMap
+from django.utils import timezone
+
+logger = logging.getLogger(__name__)
+django.setup()
+
+from django.core.management.base import BaseCommand
```

Comment:
```suggestion
import logging

from django.core.management.base import BaseCommand
from django.utils import timezone

from framework.celery_tasks import app as celery_app
from osf.models import UserSessionMap

django.setup()
logger = logging.getLogger(__name__)
```

Trivial: import order